app/src/sharedTest/java/org/oppia/android/app/devoptions/ViewEventLogsFragmentTest.kt

```diff
@@ -509,32 +398,27 @@ class ViewEventLogsFragmentTest {
   private fun logMultipleEvents() {
     oppiaLogger.logTransitionEvent(
       timestamp = TEST_TIMESTAMP,
-      eventAction = EventAction.OPEN_PROFILE_CHOOSER,
-      eventContext = null
+      eventContext = oppiaLogger.createOpenProfileChooserContext(),
     )
 
     oppiaLogger.logTransitionEvent(
       timestamp = TEST_TIMESTAMP + 10000,
-      eventAction = EventAction.OPEN_HOME,
-      eventContext = null
+      eventContext = oppiaLogger.createOpenHomeContext()
     )
 
     oppiaLogger.logTransitionEvent(
       timestamp = TEST_TIMESTAMP + 20000,
-      eventAction = EventAction.OPEN_LESSONS_TAB,
-      eventContext = oppiaLogger.createTopicContext(TEST_TOPIC_ID)
+      eventContext = oppiaLogger.createOpenInfoTabContext(TEST_TOPIC_ID)
```

Comment:
Should this be the lessons tab, instead (for parity with the previous implementation)?

Reply:
done.